package.json

```diff
@@ -49,7 +49,7 @@
     "class-autobind": "^0.1.4",
     "debounce": "^1.2.0",
     "deltachat-emoji": "^1.0.0",
-    "deltachat-node": "^0.28.0",
+    "deltachat-node": "^0.28.2",
```

Comment:
This is technically not needed since re-installing dependencies will fix it. Just saying.

Reply:
yeah we only need to do this on minor (e.g., 0.29)